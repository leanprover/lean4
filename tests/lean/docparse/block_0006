{test}
> Here's a blockquote

 with multiple paras

that ends