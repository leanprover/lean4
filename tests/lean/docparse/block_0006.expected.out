Success! Final stack:
  (Lean.Doc.Syntax.command "{" `test [] "}")
Remaining:
"> Here's a blockquote\n\n with multiple paras\n\nthat ends"